cmake/FindGSL.cmake

```diff
@@ -0,0 +1,135 @@
+# Try to find gnu scientific library GSL
```

Comment:
Why not using default script https://cmake.org/cmake/help/latest/module/FindGSL.html ?

Reply:
It looks it is present even in cmake 3.2: https://cmake.org/cmake/help/v3.2/module/FindGSL.html